src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -20,36 +21,49 @@ public class DeleteCommand extends Command {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Deletes the person identified by the index number used in the displayed person list.\n"
-            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Parameters: INDEX [OPTIONAL INDEXES] (must be a positive integer)\n"
```

Comment:
Would it be better to put "Parameters: INDEX [OPTIONAL INDEXES] (must be positive integer(s))\n" here to match the INDEXES?

Reply:
Good catch! Will update in a later commit.